tox.ini

```diff
@@ -285,9 +288,10 @@ deps =
   docker-compose==1.29.2
   requests==2.28.2
 
-  ; proto 3 and 4 tests install the respective version of protobuf
+  ; proto 3, 4 and 5 tests install the respective version of protobuf
   proto3: protobuf~=3.19.0
   proto4: protobuf~=4.0
+  proto5: protobuf~=5.26.0
```

Comment:
On line 106, you used `5.27.0`. Intentional that this is 5.26?

Reply:
Noticed that too and interpreted it as "5.26 is enough but pip install protobuf && pip freeze > test-requirements.txt got latest"